common/meta/app/src/schema/table.rs

```diff
@@ -174,7 +174,11 @@ pub struct TableMeta {
     pub engine: String,
     pub engine_options: BTreeMap<String, String>,
     pub options: BTreeMap<String, String>,
-    pub cluster_keys: Option<String>,
+    pub cluster_key: Option<String>,
+    // The vector of cluster keys.
+    pub cluster_keys: Vec<String>,
```

Comment:
`Why are here duplicated `cluster_key` and `cluster_keys`?

Reply:
cc @zhyass

Reply:
> Why are there duplicated `cluster_key` and `cluster_keys`?

The `cluster_key` records the current cluster key of the table. The `cluster_keys` records all historical cluster keys of the table.